Sources/MintKit/Mint.swift

```diff
@@ -268,6 +268,24 @@ public class Mint {
             }
         }
 
+        // copy manpages
+        do {
+            let manpages = Path("\(packageCheckoutPath)/doc/man/").glob("*")
+            let allFiles = try manpages
+                .flatMap { try $0.recursiveChildren() }
+                .map { $0.lastComponent }
+                .joined(separator: ", ")
+
+            standardOut <<< "🌱  Copying manpages for \(package.name): \(allFiles) ..."
+
+            for manpage in manpages {
+                let dest = installationPath + "../share/man/" + manpage.lastComponent
+
+                // Path#copy(_:) fails if the dest already exists.
+                try SwiftCLI.run(bash: "cp -R \"\(manpage)\" \"\(dest)\"")
```

Comment:
These files would be overwritten anytime someone does an install of any version.
Would it be better to copy them into Mint's directory to make them accessible from there, and then on global installs create symlinks

Reply:
I just recognized that homebrew symlinks manpages as well. I can do the fix, but what about `Package.resources` ? Looks like they are just copied as well.

Reply:
Not sure what you mean by `Package.resources`. They are copied into the `mintFilePath` not the `installationPath`

Reply:
Ah ok I misunderstood. Please forget. I looked at this line and thought it was `cp resourcePath /usr/local/share/$tool/` but wasn't.🙏
```swift
try resourcePath.copy(packagePath.installPath + resource)
```